CDP4Addin/Addin.cs

```diff
@@ -311,7 +315,7 @@ public override void CTPFactoryAvailable(object CTPFactoryInst)
         /// <param name="exception">The exception that occurred.</param>
         protected override void OnError(ErrorMethodKind methodKind, Exception exception)
         {
-            Utils.Dialog.ShowError(exception, "Unexpected state in CDP4-CE.Addin " + methodKind.ToString());
+            this.Utils.Dialog.ShowError(exception, "Unexpected state in CDP4-CE.Addin " + methodKind.ToString());
```

Comment:
why remove `this.`? i think this is a Utils instance from the base class coming from NetOffice, please revert

Reply:
`this.` was not removed, but added. Probably by the beautifier.

Reply:
hahaha, good, even better, ignore my comment